prelude/README.md

```diff
@@ -111,6 +121,16 @@ instance categoryArr :: Category Prim.Function
 ($) :: forall a b. (a -> b) -> a -> b
 ```
 
+Applies a function to its argument
+
+E.g.
+
+    const $ 1 2 = const 1 2 = 1
```

Comment:
This example should use a more realistic use case, since here it would be better not to use this function at all.